src/components/Layout/Layout.test.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {FC} from 'react';
+import {Provider} from 'react-redux';
+import {Router} from 'react-router-dom';
+import {cleanup, render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import {createMemoryHistory} from 'history';
+
+import store from 'store';
+
+import Layout, {LayoutProps} from '.';
+
+const Wrapper: FC = ({children}) => <Provider store={store}>{children}</Provider>;
+
+const baseProps: LayoutProps = {
+  children: (
+    <>
+      <h1>Hello World!</h1>
+      <div id="tnb" data-testid="hash-test">
+        Hello TNB!
+      </div>
+    </>
+  ),
+};
+
+describe('Layout component', () => {
+  const scrollToFn = jest.fn();
+  const scrollIntoViewFn = jest.fn();
+
+  beforeAll(() => {
+    const portal = document.createElement('div');
+    portal.setAttribute('id', 'popover-root');
+
+    document.body.appendChild(portal);
+
+    window.scrollTo = scrollToFn;
+    window.HTMLElement.prototype.scrollIntoView = scrollIntoViewFn;
+  });
+
+  afterAll(cleanup);
```

Comment:
do we need this? I tried deleting it, and the tests still passed.

it looks like its supposed to be run automatically: https://testing-library.com/docs/react-testing-library/api#cleanup